iot/api-client/manager/resources/README.md

```diff
@@ -0,0 +1,4 @@
+# Test public certificate files
+
+The public certificates in this folder are only provided for testing and should
+not be used for registering your devices.
```

Comment:
Do we tell them how to create their own somewhere?

Reply:
Yes, added note.